Release 1.2.0

jakirkham opened the pull request.

jakirkham committed:
Release 1.2.0

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
Force `lib` instead of `lib64` for library install

Appears that CLBlast via CMake is installing libraries in `lib64` on
Linux. This forces the libraries to be installed in `lib` instead.
